src/coreclr/vm/stackwalk.cpp

```diff
@@ -3161,18 +3156,6 @@ void StackFrameIterator::PostProcessingForManagedFrames(void)
         m_frameState = SFITER_NATIVE_MARKER_FRAME;
         m_crawl.isNativeMarker = true;
     }
-#ifdef TARGET_X86
-    else if (hasReversePInvoke)
```

Comment:
If you remove this then the `hasReversePInvoke` computation above it should be removed too.